src/utils/feature-flags.ts

```diff
@@ -10,6 +10,7 @@ declare const window: any
 const defaultFlagSet: LDFlagSet = {
   'banner-text': '', // by default, there is no banner text
   'enable-digital-credentials': false,
+  'enable-non-ben-corps': false,
```

Comment:
False by default, so that browser must be able to communicate with LaunchDarkly to enable this feature. (ie, failsafe)